### ISSUE TITLE
Community bin

### ISSUE THREAD

**Issue by FalseDev** [The-Coding-Academy/tca-helper-meet#8]
## Current state
As of now the community bin channel is getting filled with trash and conversations

People are posting legitimately bad code!

A recent example is a command which converts **binary to text using an API**!

Discuss below on how we can fix this and prevent bad practices and bad code from reaching everyonw

**Comment by Iusedmy1stnamelol** [comment]
Simple, watch over the channel, like how we do all channels. We have more than 10 mods. Each of us may have different timezones, but it's rare things that happen in general that 2 mods need to be there. I watch over help channels as well. While someone else is in general ETC. Slowly just lurk over each channel.

**Comment by FalseDev** [comment]
My idea was to have a bot command *submit* their code which will be approved by anyone with the official helper role as "**good code without bad practices**" and then it'll be sent over to the community bin channel, yet another use for Coding Bot v4

**Comment by Tanishquasar** [comment]
Well to avoid conservations, we can check for every message if it contains codeblocks and if it does not, then Coding Bot v4 can dm the person and ask them to decorate it with code blocks if the code is legit.

**Comment by zeffo** [comment]
I agree with False's idea, let the people send their code for approval by helpers/staff so we don't see any really bad code there. 

We should discuss the guidelines and rubrics for approving code if we go through with this.

**Comment by zeffo** [comment]
> Well to avoid conservations, we can check for every message if it contains codeblocks and if it does not, then Coding Bot v4 can dm the person and ask them to decorate it with code blocks if the code is legit.

+1, this would be very effective. If people circumvent it by putting their messages in code blocks we can punish them with whatever staff feels is suitable
